src/components/CharacterCard.js

```diff
@@ -2,9 +2,9 @@ import "./CharacterCard.css";
 
 export default function CharacterCard() {
   return (
-    <main className="app__character-card">
-      <div>CharacterCard</div> <div>CharacterCard</div> <div>CharacterCard</div>
-      <div>CharacterCard</div>
-    </main>
+    <div className="app__character-card">
+      <p>CharacterCard</p> <p>CharacterCard</p> <p>CharacterCard</p>
+      <p>CharacterCard</p>
+    </div>
   );
```

Comment:
why do you have so many character card p tags? do you just test stuff?

Reply:
oh, somehow you commented on my outdated code. Please check `files changed` :) But yes, I used it for my static version.